lib/recurly/apple-pay.js

```diff
@@ -1,12 +1,12 @@
 import Emitter from 'component-emitter';
 import errors from '../errors';
+import {CheckoutPricing} from './pricing/checkout';
```

Comment:
Is this going to be used in one of the followup PR's?

Reply:
Yeah. Prior to writing the attachment, I'll adapt these modules to use a Pricing generic proxy

Reply:
See #394